jenkins/jobs/capm3-e2e-tests.pipeline

```diff
@@ -52,7 +50,7 @@ pipeline {
         NUM_NODES = "${NUM_NODES}"
         CAPI_VERSION = "${CAPI_VERSION}"
         CAPM3_VERSION = "${CAPM3_VERSION}"
-        EPHEMERAL_TEST="${EPHEMERAL_TEST}"
+        EPHEMERAL_TEST = "${EPHEMERAL_TEST}"
```

Comment:
Do we still need to keep this var here @adilGhaffarDev  ?

Reply:
I see this being used later in CAPM3, I think we should keep it.